agent/Dockerfile.arm32v6

```diff
@@ -1,6 +1,6 @@
 # docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
 
-FROM arm32v6/golang:1.22.6-alpine3.19
+FROM arm32v6/golang:1.23-alpine3.19
```

Comment:
AS ...

Reply:
I understand that `AS` here could enforce the pattern that we have in others Dockerfile. However, I think we could do it in another pull request, as the goal of this one is only updating the version.

What do you think?